site.json

```diff
@@ -345,6 +345,216 @@
             "title": "JavaScript Demo: DataView.setUint32()",
             "type": "js"
         },
+        "typedarrayBuffer": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/typedarray-buffer.html",
+            "fileName": "typedarray-buffer.html",
+            "title": "JavaScript Demo: TypedArray.buffer",
+            "type": "js"
+        },
+        "typedarrayByteLength": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/typedarray-bytelength.html",
+            "fileName": "typedarray-bytelength.html",
+            "title": "JavaScript Demo: TypedArray.byteLength",
+            "type": "js"
+        },
+        "typedarrayBytesPerElement": {
+            "baseTmpl": "tmpl/live-js-tmpl.html",
+            "exampleCode": "live-examples/js-examples/typedarray-bytes_per_element.html",
+            "fileName": "typedarray-bytes_per_element.html",
```

Comment:
Nit: For consistency, use `-` between words as apposed to a mix of dashes and underscores.

Reply:
Fine to do this. My perhaps obsessive reasoning was that the dash separated the type of object from the method/property and since we're dealing with property BYTES_PER_ELEMENT - I just wanted to keep consistency (albeit in lower case) with the actual property name.